src/moonlink_connectors/src/pg_replicate/conversions/text.rs

```diff
@@ -305,7 +339,25 @@ impl TextFormatConverter {
             Type::OID_ARRAY => {
                 TextFormatConverter::parse_array(str, |str| Ok(Some(str.parse()?)), ArrayCell::U32)
             }
-            _ => Err(FromTextError::InvalidConversion()),
+            _ => match typ.kind() {
+                Kind::Composite(fields) => TextFormatConverter::parse_composite(str, fields),
+                Kind::Array(inner_type) => {
+                    // Check if the array contains composite types.
+                    // PostgreSQL supports multi-dimensional arrays (e.g., int[][], text[][]),
+                    // but here we currently only handle arrays of composite types.
+                    match inner_type.kind() {
+                        Kind::Composite(fields) => {
+                            TextFormatConverter::parse_composite_array(str, fields)
+                        }
+                        Kind::Array(_) => {
+                            // TODO: Multi-dimensional arrays not yet implemented
+                            Err(FromTextError::InvalidConversion())
```

Comment:
Need to print out the unsupported type (included in error type), otherwise hard to debug

Reply:
Not requested for the current PR (since it's not a regression), feel free to leave a TODO and an issue so we don't forget.